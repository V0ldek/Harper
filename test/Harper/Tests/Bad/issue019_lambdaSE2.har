main :: sideeffect -> ();
main = {
  (f :: Integer -> Integer) = (\(x :: Integer) => {
      eval printLn (f 10) ();
      return (\(n :: Integer) => n * x);
      }) 42 ();

  eval printLn (f 10) ();
};