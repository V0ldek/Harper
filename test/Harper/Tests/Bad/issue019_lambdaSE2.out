Error: undeclared identifier `f`.
During evaluation of:
 f
Located at line 4 column 21

Execution terminated with an error: type error.